### ISSUE TITLE
Null ref: T2.ScriptedSequence_RescueKittenAndReceiveBlessing.Update / UnityEngine.Transform.get_position

### ISSUE THREAD

**Issue by ChrisNZL** [ChrisNZL/Tallowmere2#758]
Auto report. 0.2.0.

Feedback ID: 20201207-L6249

```
Update: Revealing room: Room 5: Sewers
4:16:11, Frame 36436, LOG »  ScriptedSequenceStarter.OnCreaturePressedActionToInteract: Telling server to start sequence

4:16:11, Frame 36442, EXCEPTION »  NullReferenceException
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

UnityEngine.Transform.get_position ()
T2.ScriptedSequence_RescueKittenAndReceiveBlessing.Update ()

GameStates: InitialisingAsNetworkHost, RompingThroughDungeon, ShowingCutscene
GameSetupMode: NetworkCoop
Players: 2
DungeonRoomTransitionerState: Null
Room: 5 / 5
RoomModifiers: None
YouHaveDiedState: Null
SystemPlayer InputDevice: Keyboard
HumanPlayer 1 InputDevice: Keyboard
```

**Comment by ChrisNZL** [comment]
Additional null ref somewhere within `ScriptedSequence_RescueKittenAndReceiveBlessing.Update`...

Received three error reports at the same time. Same online game.

```
3:26:51, Frame 54202, EXCEPTION »  NullReferenceException
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

T2.ScriptedSequence_RescueKittenAndReceiveBlessing.Update ()

GameStates: InitialisingAsNetworkHost, RompingThroughDungeon, ShowingCutscene
OnlineGameInfo: Device is online.
Server: Lightning-Beijing-01
GameID: L69DH
GameSetupMode: NetworkCoop
Players: 3
DungeonRoomTransitionerState: Null
Room: 5 / 5
RoomModifiers: None
YouHaveDiedState: Null
SystemPlayer InputDevice: XInput Controller
HumanPlayer 1 InputDevice: XInput Controller
```

**Comment by ChrisNZL** [comment]
Additional auto report, 0.2.2a. Feedback ID: 20210108-X8LFD

From the logs and savefiles received, player is duplicating and manipulating their savefiles (editing the binary data I assume), causing the limit to be reached.

Still need to handle the limit more gracefully.

**Comment by ChrisNZL** [comment]
Should be fixed in 0.2.3:

- Added many null checks within `ScriptedSequence_RescueKittenAndReceiveBlessing.Update`.

- Now, with Online Co-op, if a cutscene is in progress, damage messages are no longer processed on the server – believe this was the underlying issue.
